packages/react-components/react-toast/src/components/Toaster/useFocusManagement.ts

```diff
@@ -0,0 +1,70 @@
+import { isHTMLElement } from '@fluentui/react-utilities';
+import * as React from 'react';
+import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
+import { ArrowDown, ArrowUp } from '@fluentui/keyboard-keys';
+import { toastContainerClassNames } from '../ToastContainer';
+
+/**
+ * @internal
+ * Toasts  should be arrow navigable and focus should be trapped in a stack of tasts
+ * This is a temporary measure, Tabster does not have an API yet to enable mover arrow keys from within grouppers
+ * Once tabster fully supports this use case, remove this hook
+ */
+export function useFocusManagement_unstable() {
```

Comment:
Can you please prefix name with `Toaster` i.e. `useToaster`, so if someone will export this accidentally we will not have naming collisions.